docs/src/docs/use-cases.md

```diff
@@ -1,121 +1,86 @@
 
-Weyland-Yutani Corporation has been adding AI and ML models to its portfolio for both internal- and customer-facing deployment. There are three main groups that are involved in the development lifecycle for their AI/ML work:
-* AI/ML Team, composed of data scientists, data engineers, and MLOps engineers. This specialized group builds, tunes, and validates the models that increasingly touch every aspect of the organization's work.
-* Application Teams, composed of software and hardware engineers. This group builds the applications that the business and customers interact with - over time more and more of these applications are being integrated into the models the AI/ML team works on.
-* SRE Team, composed of infrastructure, pipeline, and release specialists who make sure the models and applications are deployed quickly and safely. They also monitor all production applications to ensure security, correctness, availability, and performance.
+Here’s how we used ModelKits along with the tools we already used in our data science, software development, and infrastructure management teams.
 
-There has been a heated argument within Weyland-Yutani about how best to manage the collaboration, testing, and deployment nuances of AI/ML-integrated applications. Today, this is a friction-filled process that relies on significant manual effort from every team. Each team is often having to ask for help from other teams to prepare models for the work each needs to do, slowing down progress and introducing needless human errors.
+### Using Tags
 
-...but all that's about to change! (Hint: the change involves KitOps...surprise!)
+We used ModelKit tag names to give team members a quick way to determine where an AI project was in its lifecycle and whether they needed to get involved. Since ModelKits are immutable but use content-addressable storage, two ModelKits with the same contents but different tags only result in one ModelKit in storage (with two tag "pointers").
 
-## 1/ Collaborating on an AI-Enabled Application for Production
+* _tuning_: dataset is designed for model tuning [1]
+* _validating_: dataset is designed for model validation [1]
+* _trained_: model has completed training phase [2]
+* _tuned_: model has completed fine-tuning phase [2]
+* _challenger_: model should be prepared to replace the current champion model [3]
+* _champion_: model is deployed in production [4]
+* _rollback_: the model was the previous "champion" model, before it was replaced by the "challenger" [5]
+* _retired_: the model is no longer appropriate for production usage [6]
 
-### AI/ML Team Collaboration
+[1] Used for ModelKits that contained only datasets. These were used almost exclusively by the data science team.
```

Comment:
Is this section meant to be one big paragraph? In my preview it wraps as a single paragraph